forms-flow-api/src/formsflow_api/models/process.py

```diff
@@ -136,3 +137,13 @@ def get_latest_version(cls, process_name):
         )
 
         return query
+
+    @classmethod
+    def fetch_histories_by_process_name(cls, process_name: str) -> List["Process"]:

```

Comment:
List["Process"]:
Having the Process class as type added is better than as that would make the typing, string value is for forward reference when you don't have access to the class yet

Reply:
updated code